monkestation/code/modules/botany/components/plant_growing.dm

```diff
@@ -139,7 +139,7 @@
 
 /datum/component/plant_growing/proc/try_plant_seed(datum/source, obj/item/seeds/seed, mob/living/user)
 	SIGNAL_HANDLER
-	if(istype(seed, /obj/item/storage/bag/plants))
+	if(istype(seed, /obj/item/storage/bag/plants) && !istype(seed, /obj/item/seeds/sample))
```

Comment:
wait, (sub)types of `/obj/item/storage/bag/plants` will never be a (sub)type of `/obj/item/seeds/sample` anyways?